.env

```diff
@@ -12,5 +12,5 @@ APP_URL=http://apptemplate.localhost
 #
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
 # DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
-DATABASE_URL=mysql://root:root@mysql:3306/apptemplate
+DATABASE_URL=mysql://root:root@db:3306/apptemplate
```

Comment:
I would recommend not changing the defaults in the .env but rather change the ENV var in GH workflow

Reply:
Ok, renamed the service from `db` to `mysql`, so no need for overriding the environment in GH workflow.